BlazorUI/Startup.cs

```diff
@@ -0,0 +1,63 @@
+using System;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5088867)